### PR TITLE
Feature/ted 1219 packages

### PR DESCRIPTION
**Pull request by kaleanych**

### CODE REVIEW

File: ted_sws/notice_packager/model/metadata.py [hunk 1]
```diff
@@ -35,38 +53,52 @@
...
-ACTION_UPDATE = "update"
-ACCEPTED_ACTIONS = [ACTION_CREATE, ACTION_UPDATE]
-
 REVISION = "0"
 
 
-def validate_notice_action_type(v):
-    if v not in ACCEPTED_ACTIONS:
-        raise ValueError('No such action: %s' % v)
+def validate_mets_type(v):
```
Comment (Dragos0000) [inline ted_sws/notice_packager/model/metadata.py:59]: what is v ? victory ?

File: ted_sws/notice_packager/model/metadata.py [hunk 2]
```diff
@@ -35,38 +53,52 @@
...
+    tmd_othermdtype: str = METS_TMD_OTHERMDTYPE
+    file_id: Optional[str]
+    notice_file_href: Optional[str]
+    notice_file_mimetype: Optional[str] = METS_NOTICE_FILE_MIMETYPE
+    notice_file_checksum: Optional[str]
+    notice_file_checksum_type: Optional[str] = METS_NOTICE_FILE_CHECKSUM_TYPE
+
+    @validator('type')
+    def validate_notice_action_type(cls, v):
+        validate_mets_type(v)
```
Comment (Dragos0000) [inline ted_sws/notice_packager/model/metadata.py:100]: what is v ? vicar?

**Review by Dragos0000 — APPROVED** [review]
tell us more about v

File: ted_sws/notice_packager/model/metadata.py (hunk 1 above)
Comment (kaleanych) [reply ted_sws/notice_packager/model/metadata.py:59]: it’s a short name for value when used in a very small context (likke: k => v: key => value)

**Pull request merged**